.flake8

```diff
@@ -0,0 +1,42 @@
+[flake8]
+ignore = E,W,F,N
+
+select =
+    E101
+    E128
+    E131
+    E201
+    E202
+    E203
+    E225
+    E226
+    E231
+    E241
+    E261
+    E262
+    E265
+    E271
+    E4
+    E7
+    E9
+    W291
+    W292
+    W293
+    W6
+    #F401
```

Comment:
Just fyi, I want to reactivate this one eventually because it flags unused imports. I commented it out for now because it complains about the imports in one of the `__init__.py` files and I haven't figured out yet how to fix this.